src/xdocs/google_style.xml

```diff
@@ -1485,6 +1501,29 @@
                     test</a>
                 </td>
               </tr>
+              <tr>
+                <td><a name="4.8.5.5"/>
+                  <a href="#4.8.5.5">
+                    <span class="wrapper inline">
+                      <img src="images/anchor.png" alt=""/>
+                    </span>
+                  </a>
+                </td>
+                <td>
+                  <a href="styleguides/google-java-style-20220203/javaguide.html#s4.8.5.5-local-parameter-annotation-style  ">
+                    4.8.5.5 Parameter and local variable annotations
+                  </a>
+                </td>
+                <td>
+                  --
+                  <br/>
+                  For type-use annotations see
```

Comment:
you need double BR
example https://github.com/checkstyle/checkstyle/blob/5433ccf2e00802dc896c8f82422c4ba565707528/src/xdocs/google_style.xml#L1804-L1805

you can check website rendering on your local - https://github.com/checkstyle/checkstyle/wiki/How-to-run-certain-phases-and-validations#how-to-generate-website-only